sources/packages/backend/apps/api/src/route-controllers/education-program/education-program.controller.service.ts

```diff
@@ -114,6 +115,11 @@ export class EducationProgramControllerService {
         if (error.name === EDUCATION_PROGRAM_NOT_FOUND) {
           throw new NotFoundException(error.message);
         }
+        if (error.name === DUPLICATE_SABC_CODE) {
```

Comment:
Usually, we include all possible generated exceptions from the "body" of the endpoint or resulting from its processing, in the Swagger decorators, as shown below. The idea would be provided to someone looking to swagger an idea about all possible scenarios where an exception would happen.
```ts
@ApiUnprocessableEntityResponse({
    description:
      "Not able to a save the program due to an invalid request or " +
      "SABC code is duplicated.",
  })
```

Reply:
Ok! Thanks! Added `not able to find the education program` too.

Reply:
ok. Done. Thanks!